ansible/roles/nextcloud/tasks/main.yml

```diff
@@ -1,11 +1,8 @@
 ---
 
 - name: Install nextcloud client
```

Comment:
Let's remove the nextcloud client entirely as this has been a pain on my systems. :-)